x-pack/plugins/observability/public/utils/slo/labels.ts

```diff
@@ -124,3 +114,34 @@ export function toDurationLabel(durationStr: string): string {
       });
   }
 }
+
+export function toDurationAdverbLabel(durationStr: string): string {
```

Comment:
Should we add `capitalize` as a second argument. Then have translations for both lower case and capitalized?

Reply:
I just capitalized everything by default. We might need to introduce one translation key per duration instead of composing them actually...